plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherPlugin.kt

```diff
@@ -120,6 +127,17 @@ class PlayPublisherPlugin : Plugin<Project> {
                 doFirst { logger.warn("$name is deprecated, use ${publishListingTask.get().name} instead") }
             }
 
+            val publishProductsTask = project.newTask<PublishProducts>(
+                    "publish${variantName}Products",
+                    "Uploads all Play Store in-app products for $variantName."
+            ) {
+                init()
+                productsDir = File(playResourcesTask.get().resDir, PRODUCTS_PATH)
```

Comment:
Let's move this into a lazy block inside the task as in #354

Reply:
👍